internal/app/vodtester/vodtester_app.go

```diff
@@ -128,12 +128,16 @@ func (vt *vodTester) Start(fileUrl string, taskPollDuration time.Duration) (int,
 		task, err := vt.lapi.GetTask(exportTask.ID)
 		if err != nil {
 			glog.Errorf("Error retrieving task id=%s err=%v", exportTask.ID, err)
-			return 246, fmt.Errorf("Error retrieving task id=%s: %w", exportTask.ID, err)
+			return 246, fmt.Errorf("error retrieving task id=%s: %w", exportTask.ID, err)
 		}
-		if task.Output.Export.IPFS.VideoFileGatewayUrl != "" {
+		if task.Output != nil && task.Output.Export != nil && task.Output.Export.IPFS != nil && task.Output.Export.IPFS.VideoFileGatewayUrl != "" {
 			glog.Infof("Export success, task id=%s assetId=%s ipfs link=%s", exportTask.ID, exportTask.InputAssetID, task.Output.Export.IPFS.VideoFileGatewayUrl)
 			break
 		}
+		if task.Status.Phase == "completed" {
+			glog.Errorf("Error exporting asset task id=%s", exportTask.ID)
+			return 247, fmt.Errorf("error export failed, task id=%s assetId=%s", exportTask.ID, exportTask.InputAssetID)
+		}
```

Comment:
Is there any use in accounting for a default case here? What happens if nothing goes wrong?